<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>511</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>129</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>129</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>129</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>129</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>661</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_hubs">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hubs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Routes">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Routes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_MailItems">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mail Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Companys">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Companies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Reports">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reports</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Users/Pricetech/Desktop/SWEN301/logo_small.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdd_Hub">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAdd_Hub"/>
    <addaction name="actionAdd_Route"/>
    <addaction name="actionAdd_Mail_Item"/>
    <addaction name="actionAdd_Company"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuAdd_Hub"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Hub">
   <property name="text">
    <string>Hubs</string>
   </property>
  </action>
  <action name="actionAdd_Route">
   <property name="text">
    <string>Routes</string>
   </property>
  </action>
  <action name="actionAdd_Mail_Item">
   <property name="text">
    <string>Mail Items</string>
   </property>
  </action>
  <action name="actionAdd_Company">
   <property name="text">
    <string>Companys</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
